interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/mlmodel/'
+  );
+}])
```

Comment:
This is such a small component, I think it would be okay to have the resource as part of `mlModelComponents`. There is nothing technically wrong with having this factored out and I'd say it should be required if we needed to break up a larger module but there is also something to be said for having fewer files to flip through.

I guess I'd call this a "weak" recommendation to add this factory to `mlModelComponents`. Feel free to ignore if you don't agree.